src/Console/Command.php

```diff
@@ -89,6 +90,29 @@ public function getLoader()
     }
 
     /**
+     * Set the suite runner used by the Peridot command.
+     *
+     * @param RunnerInterface $runner
+     * @return $this
+     */
+    public function setRunner(RunnerInterface $runner)
+    {
+        $this->runner = $runner;
+        return $this;
+    }
+
+    /**
+     * Return the runner used by the Peridot command. Defaults to
+     * an instance of Peridot\Runner\Runner.
+     *
+     * @return Runner
+     */
+    public function getRunner()
+    {
+        return $this->runner;
```

Comment:
If we can set any RunnerInterface, then we can return more than just Runner (fix typehint).  Same holds true for the constructor.  It can just be Runner Interface instead of Peridot\Runner\Runner.